metricbeat/module/elasticsearch/node_stats/data_test.go

```diff
@@ -0,0 +1,35 @@
+// +build !integration
+
+package node_stats
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
don't use an underscore in package name